looprpc/client.proto

```diff
@@ -372,4 +372,11 @@ message QuoteResponse {
     An estimate of the on-chain fee that needs to be paid to sweep the HTLC.
     */
     int64 miner_fee = 3;
+    
+    /**
+    If an error happens during on-chain fee estimation in lnd, this contains
+    the error. The client should decide if it's ok to continue (i.e. external
+    loop in) or not.
```

Comment:
I am wondering what the user expects in a quote response when the intention is to use an external address. A zero miner fee? Or a happy case fee (1 inputs, 1 htlc out, 1 change out)?

How about passing in an `external` bool to the quote request to skip the lnd wallet fee estimation? It makes it all a bit more transparent. Returning an error sometimes for an external funded swap quote may be confusing.

Reply:
Good idea! I like that much more than passing an error back.
I wasn't sure if it was OK to just skip the fee estimation completely on an external swap in.